app/collectors/lambda.scala

```diff
@@ -74,8 +76,10 @@ case class Lambda(
   region: String,
   runtime: String,
   tags: Map[String, String],
+  override val app: List[String],
```

Comment:
I have used `List[String]` (rather than `Option[String]`) for consistency with the `instance` model. 

I suspect that this choice of type may be a legacy thing, so I briefly attempted to refactor and use `Option[String]` for both. However, we are already exposing this field to clients as an array (e.g. `"app":["mobile-sport"]`), so I don't think we can easily update this type.